e2e/cephfs.go

```diff
@@ -402,6 +406,80 @@ var _ = Describe(cephfsType, func() {
 				}
 			})
 
+			By("create a PVC and check PVC/PV metadata on CephFS subvolume", func() {
+				err := createCephfsStorageClass(f.ClientSet, f, true, nil)
+				if err != nil {
+					e2elog.Failf("failed to create CephFS storageclass: %v", err)
+				}
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					e2elog.Failf("failed to load PVC: %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC: %v", err)
+				}
+
+				subvol, err := listCephFSSubVolumes(f, fileSystemName, subvolumegroup)
+				if err != nil {
+					e2elog.Failf("failed to list CephFS subvolumes: %v", err)
+				}
+
+				pvcName, stdErr, err := execCommandInToolBoxPod(f,
+					formatSubvolumeMetaGetCmd(fileSystemName, subvol[0].Name, subvolumegroup, pvcNameKey),
+					rookNamespace)
+				if err != nil || stdErr != "" {
+					e2elog.Failf("failed to get PVC name %s/%s %s: err=%v stdErr=%q",
+						fileSystemName, subvol[0], pvcNameKey, err, stdErr)
+				}
+				pvcName = strings.TrimSuffix(pvcName, "\n")
+				if pvcName != pvc.Name {
+					e2elog.Failf("expected pvcName %q got %q", pvc.Name, pvcName)
+				}
+
+				pvcNamespace, stdErr, err := execCommandInToolBoxPod(f,
```

Comment:
move this to a helper function as we need to do same validation for PVC restore and PVC clone tests